cpp/client.cc

```diff
@@ -98,3 +98,21 @@ void Client::monitor(std::string username,
     }
   }
 }
+
+void Client::stream(std::string hashtag,
+                     std::unique_ptr<chirp::ServiceLayer::Stub>& stub_) {
+  grpc::ClientContext context;
+  chirp::StreamRequest request;
+  request.set_hashtag(hashtag);
+  chirp::StreamReply reply;
+  std::unique_ptr<grpc::ClientReader<chirp::StreamReply> > stream_handle(
+      stub_->stream(&context, request));
+  while (true) {
```

Comment:
You should describe the termination condition, since it's not apparent.

Reply:
added a comment